examples/kingworld/README.md

```diff
@@ -0,0 +1,15 @@
+# KingWorld Starter
+
+## Scaffold
+To get start with this template, simply paste the command in to your terminal:
+```bash
+bun create kingworld
```

Comment:
Please change this to ``bun create kingworld ./kingworld-example`` to emphasize that the user can choose their own folder.